TestAssets/TestProjects/TestProjectToProjectDependencies/src/L21/project.json

```diff
@@ -1,10 +1,10 @@
 {
   "version": "1.0.0-*",
   "dependencies": {
-    "NETStandard.Library": "1.5.0-rc2-24018"
+    "Microsoft.NETCore.App": "1.0.0-rc2-*"
   },
   "frameworks": {
-    "netstandardapp1.5": {
+    "netcoreapp1.0": {
```

Comment:
Should be netstandard right?  L1 and L2 projects are libraries and should be netstandard.